jefferies_tube.gemspec

```diff
@@ -24,6 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec", '~> 3.0'
 
-  spec.add_dependency "bundler-audit", '~>0.5.0'
```

Comment:
so then this version has a hard dependency on some version of bundler? or is 0.6.1 just hard requiring a higher version of bundler?

is this worth a callout in the changelog?

Reply:
Yeah this was kinda weird. Codeship was failing because it was using bundler version 2, but every bundler version before 0.6.1 requires bundler ~> 1.2

So yeah I'll put something in the changelog.

Reply:
I think this is a breaking change?

Reply:
does 0.6.1 require bundler 2? or is it fine with `~> 1.2`? it doesn't feel like a thing that deserves a major version bump

Reply:
https://rubygems.org/gems/bundler-audit/versions/0.6.1

Reply:
It requires >= 1.2. So this might not be breaking. That's what I'm kinda confused on.

Reply:
This actually seems the opposite. This seems non breaking?

Reply:
yea i think that's true. i believe it's non breaking